src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -540,6 +540,12 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
 ::v-deep #certify-section {
   .container {
     padding-right: 4px !important;
+
+    .v-input--checkbox {
+      .v-input__slot {
+        align-items: flex-start;
+      }
+    }
```

Comment:
No action required here but another option to control the nested styling is to slot the component itself like such. 
```
<v-checkbox  
 <span` slot="label" class="<apply-style-class-here>">
    Checkbox label
 </span>
</v-checkbox>
```

Reply:
Since its a shared component I don't want to change any styling inside it

Reply:
oo right, sorry i didn't notice that :) Good call then!